scheduler/src/cook/mesos/util.clj

```diff
@@ -24,7 +24,56 @@
             [metatransaction.core :refer (db)]
             [metrics.timers :as timers]
             [plumbing.core :as pc :refer (map-vals map-keys)])
-  (:import [java.util Date]))
+  (:import
+    [com.google.common.cache Cache]
+    [com.google.common.cache CacheBuilder]
```

Comment:
Can we combine these two onto a single line?

Reply:
Done

Reply:
They still seem to be on separate lines

Reply:
Hadn't pushed when you saw that. Was waiting to roll up feedback and push all at once.